arduino/WandCore/LEDClient.h

```diff
@@ -1,18 +1,21 @@
-/*
-  LED Client
-
-  Manages a common cathode RGB LED.
-*/
-
-#ifndef LED_CLIENT
-#define LED_CLIENT
-
-#include "Arduino.h"
-
-// Initializes RGB pins.
-void setupRGBLED(int redPin, int greenPin, int bluePin);
-
-// Sets the LED to an RGB color.
-void setRGBLED(int r, int g, int b);
-
-#endif
+/*
+  LED Client
+
+  Manages a common cathode RGB LED.
+*/
+
+#ifndef LED_CLIENT
+#define LED_CLIENT
+
+#include "Arduino.h"
+
+// Initializes RGB pins.
+void setupRGBLED(int redPin, int greenPin, int bluePin);
+
+// Sets the LED to an RGB color.
+void setRGBLED(int r, int g, int b);
+
+// Sets the LED to the next RGB color in the colorloop.
+void incrementRGBColorloop();
+

```

Comment:
Do you think this begs for a `resetRGBColorloop()` to reset the colorloop state variables?